.github/workflows/build_and_test.yml

```diff
@@ -63,6 +73,8 @@ jobs:
             --spawn_strategy=local \
             -c dbg \
             --strip="never" \
+            --remote_cache=$GOOGLE_REMOTE_CACHE \
+            --google_credentials=$GOOGLE_APPLICATION_CREDENTIALS \
             :eventuals
```

Comment:
Can we also add `--verbose_failures`? If the credentials stop working at some point (or just aren't there) `bazel` will give a very limited error, with `--verbose_failures` it's much clearer.